website/docs/visual-testing/methods.md

```diff
@@ -0,0 +1,265 @@
+---
+id: methods
+title: Methods
+---
+
+The following methods are added to the Global WebdriverIO `browser`-object.
+
+## Save Methods
+
+:::info TIP
+Only use the Save Methods when you **don't** want to compare screens, but only want to have an element-/screenshot.
+:::
+
+### `saveElement`
+
+#### Usage
+
+```ts
+await browser.saveElement(
+    // element
+    "#element-selector",
```

Comment:
Can you also pass in a WebdriverIO element?

Reply:
Yes, will change this